lutris/gui/lutriswindow.py

```diff
@@ -66,6 +66,8 @@ def __init__(self, application, **kwargs):
         width = int(settings.read_setting('width') or 800)
         height = int(settings.read_setting('height') or 600)
         self.window_size = (width, height)
+        self.maximized  = settings.read_setting('maximized') == 'True'
```

Comment:
When saving a bool to a file, it gets retrieved as string. Running `bool('False')` returns True, so `bool()` won't work.